xarray/core/alignment.py

```diff
@@ -367,6 +391,7 @@ def _broadcast_dataset(ds):
 
 
 def broadcast_arrays(*args):
+    import warnings
```

Comment:
oops! this is probably an indicator that nobody is relying on this deprecated functionality, though ;).